src/components/drawer/index.tsx

```diff
@@ -97,10 +97,6 @@ interface Props {
    * Callback for just before right item full swipe
    */
   onWillFullSwipeRight?: Function;
-  /**
-   * Haptic trigger function to use onToggleSwipeLeft
-   */
-  leftToggleHapticTrigger?: Function;
```

Comment:
This is a breaking change. You can't just delete props, you need to deprecate them. Give the user a chance to migrate.

Reply:
Right, adding it to the V7 spreadsheet